src/Microsoft.NET.Sdk.Functions/Microsoft.NET.Sdk.Functions.csproj

```diff
@@ -1,18 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.5;net46</TargetFrameworks>
-    <PackageVersion>1.0.2</PackageVersion>
+    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
```

Comment:
The task assembly should still target netstandard1.5 so that core msbuild can still load it. The NuGet package that gets generated for this task assembly (Functions sdk) should add package dependencies to the 3.0 webjobs packages (which are netstandard2.0 compatible.)